nullaway/src/test/java/com/uber/nullaway/tools/SerializationTestHelper.java

```diff
@@ -120,17 +118,27 @@ private void compareFixes(FixDisplay[] outputFixDisplays) {
     BufferedReader reader;
     try {
       reader = Files.newBufferedReader(this.outputPath, Charset.defaultCharset());
+      String header = reader.readLine();
+      if (!header.equals(SuggestedFixInfo.header())) {
+        fail(
+            "Expected header of fixes.tsv to be: "
+                + SuggestedFixInfo.header()
+                + "\bBut found: "
```

Comment:
`\b`? Why backspace here?

Reply:
I actually meant `\n`, fixed it. Thank you for noticing it.